src/main/java/uk/gov/digital/ho/pttg/application/util/DiacriticNameNormalizer.java

```diff
@@ -15,17 +28,37 @@ public Individual normalizeNames(Individual individual) {
     }
 
     private String toBasicLatin(String name) {
-        String nameWithoutAccents = StringUtils.stripAccents(name);
-        return permitOnlyBasicLatinCharacters(nameWithoutAccents);
+        String replacedCharactersFromMapping = replaceCharactersFromMapping(name);
+        String nameWithoutAccents = StringUtils.stripAccents(replacedCharactersFromMapping);
+        return removeNonBasicLatinCharacters(nameWithoutAccents);
+    }
+
+    private String replaceCharactersFromMapping(String name) {
+        if (name == null) {
+            return null;
+        }
+
+        StringBuilder stringBuilder = new StringBuilder(name.length());
+        for (char character : name.toCharArray()) {
```

Comment:
Might be cleaner with streams, then again might not - worth considering if you haven't already.

Reply:
Had a quick look at changing this, but there is no easy way (that I can see) for changing a char array to a stream. Would need to put into a list first, so I think I prefer it with the loop